SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -2,12 +2,15 @@ package com.jetbrains.handson.mpp.mobile
 
```

Comment:
This Presenter file is now quite large, would be worth thinking about pulling some of the behaviour out. Perhaps having a file that deals with querying the API and a file that deals with DateTime handling/formatting.